src/core/src/sim/mc/moves/rotate.rs

```diff
@@ -20,41 +22,38 @@ pub struct Rotate {
     molid: usize,
     /// New positions of the atom in the rotated molecule
     newpos: Vec<Vector3D>,
-    /// Normal distribution, for generation of the axis
-    axis_rng: Normal,
-    /// Range distribution, for generation of the angle
-    angle_rng: Range<f64>,
+    /// Displacement
+    displacement: Displacement,
 }
 
 impl Rotate {
     /// Create a new `Rotate` move, with maximum angular displacement of `theta`,
     /// rotating all the molecules in the system.
-    pub fn new(theta: f64) -> Rotate {
-        Rotate::create(theta, None)
+    pub fn new(theta: f64, update_freq: u64, target_acceptance: f64) -> Rotate {
```

Comment:
I see. You would then create a new move and set everything explicitly before adding it to the propagator?